index.js

```diff
@@ -0,0 +1,74 @@
+//CALCULATIONS
+
+//UNIT1
```

Comment:
We should remove these comments here.
Remember that others will be adding more functions on top of your code, so it Unit 1 and Unit 2 won't make sense in the context. 
You already have descriptive function and variables name, so you don't those comments here